core/src/replay_stage.rs

```diff
@@ -9023,6 +9047,7 @@ pub(crate) mod tests {
             &poh_recorder,
             &leader_schedule_cache,
             &rpc_subscriptions,
+            &None,
```

Comment:
Can we add/modify a test that provides a `SlotStatusNotifier` to get some coverage?

Reply:
Will handle it in another PR.